File: Paradigmas/Atv-09_04_exercicio/LojaVeiculos.java
import java.util.Scanner;

public class LojaVeiculos {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // Recebe o valor do veículo
        System.out.print("Digite o valor do veículo (R$): ");
        float valorVeiculo = scanner.nextFloat();

        if (valorVeiculo <= 0 || valorVeiculo > 200000) {
            System.out.println("Valor inválido.");
            return;
        }

        // Identifica a categoria
        int categoria;
        if (valorVeiculo <= 80000)
            categoria = 1;
        else if (valorVeiculo <= 120000)
            categoria = 2;
        else if (valorVeiculo <= 140000)
            categoria = 3;
        else if (valorVeiculo <= 160000)
            categoria = 4;
        else
            categoria = 5;

        System.out.println("Categoria do veículo: " + categoria);

        // Escolhe a forma de pagamento
        System.out.println("1 - À vista (10% de desconto)");
        System.out.println("2 - Entrada de 50% (juros de 5% a.a. em 4 anos)");
        System.out.println("3 - Entrada de 25% (juros de 8% a.a. em 4 anos)");
        System.out.print("Escolha a forma de pagamento: ");
        int opcao = scanner.nextInt();

        if (opcao < 1 || opcao > 3) {
            System.out.println("Opção inválida.");
            return;
        }

        // Calcula o valor final
        float valorFinal, entrada, restante, juros, parcela;

        if (opcao == 1) {
            valorFinal = valorVeiculo * 0.90f;
            System.out.printf("Valor final à vista: R$ %.2f\n", valorFinal);

        } else if (opcao == 2) {
            entrada = valorVeiculo * 0.50f;
            restante = valorVeiculo * 0.50f;
            juros = restante * 0.05f * 4;
            valorFinal = entrada + restante + juros;
            parcela = (restante + juros) / 48;
            System.out.printf("Entrada: R$ %.2f\n", entrada);
            System.out.printf("48x de R$ %.2f\n", parcela);
            System.out.printf("Total: R$ %.2f\n", valorFinal);

        } else {
            entrada = valorVeiculo * 0.25f;
            restante = valorVeiculo * 0.75f;
            juros = restante * 0.08f * 4;
            valorFinal = entrada + restante + juros;
            parcela = (restante + juros) / 48;
            System.out.printf("Entrada: R$ %.2f\n", entrada);
            System.out.printf("48x de R$ %.2f\n", parcela);
            System.out.printf("Total: R$ %.2f\n", valorFinal);
        }

        scanner.close();
    }
}

File: Paradigmas/Atv-09_04_exercicio/loja_veiculos.py
# Recebe o valor do veículo
valor_veiculo = float(input("Digite o valor do veículo (R$): "))

if valor_veiculo <= 0 or valor_veiculo > 200000:
    print("Valor inválido.")
    exit()

# Identifica a categoria
if valor_veiculo <= 80000:
    categoria = 1
elif valor_veiculo <= 120000:
    categoria = 2
elif valor_veiculo <= 140000:
    categoria = 3
elif valor_veiculo <= 160000:
    categoria = 4
else:
    categoria = 5

print(f"Categoria do veículo: {categoria}")

# Escolhe a forma de pagamento
print("1 - À vista (10% de desconto)")
print("2 - Entrada de 50% (juros de 5% a.a. em 4 anos)")
print("3 - Entrada de 25% (juros de 8% a.a. em 4 anos)")

opcao = int(input("Escolha a forma de pagamento: "))

if opcao < 1 or opcao > 3:
    print("Opção inválida.")
    exit()

# Calcula o valor final
if opcao == 1:
    valor_final = valor_veiculo * 0.90
    print(f"Valor final à vista: R$ {valor_final:.2f}")

elif opcao == 2:
    entrada = valor_veiculo * 0.50
    restante = valor_veiculo * 0.50
    juros = restante * 0.05 * 4
    valor_final = entrada + restante + juros
    parcela = (restante + juros) / 48
    print(f"Entrada: R$ {entrada:.2f}")
    print(f"48x de R$ {parcela:.2f}")
    print(f"Total: R$ {valor_final:.2f}")

elif opcao == 3:
    entrada = valor_veiculo * 0.25
    restante = valor_veiculo * 0.75
    juros = restante * 0.08 * 4
    valor_final = entrada + restante + juros
    parcela = (restante + juros) / 48
    print(f"Entrada: R$ {entrada:.2f}")
    print(f"48x de R$ {parcela:.2f}")
    print(f"Total: R$ {valor_final:.2f}")
